specification/devcenter/data-plane/Microsoft.DevCenter/stable/2023-04-01/devcenter.json

```diff
@@ -2760,24 +2745,13 @@
         ]
       }
     },
-    "EnvironmentUpdateProperties": {
```

Comment:
@drielenr does the C# SDK you already released reference this type anywhere?

Reply:
Nope, this is not a model at all in the sdk - we just add `parameters` to Environment model

Reply:
We do can get rid of it if we want to